app/retail/emails.py

```diff
@@ -533,14 +533,15 @@ def render_new_bounty_roundup(to_email):
 Hi there,
 </p>
 <p>
-We're excited to formally announce an expansion of our ECF Partnership, bringing $60K in funding to the <a href="https://medium.com/@mitchkosowski/cbe1d2d1c071">Ethereum Infrastructure layer</a>!
-As stated in the article, if you know of any open source projects / specific Github issues which could use some funding, <a href="https://gitcoin.co/requests">make a Gitcoin Request</a> and comment 'ECF'.
-We're very excited to make it easy to contribute to the ecosystem, while getting paid.
+Hacktoberfest is a month long celebration of open source. As Gitcoiners, we support anything that grows open source. Next Monday, we're going to formally
+announce our plans for Hacktoberfest...but you can see them <a href="https://medium.com/p/98825f199af2/edit">here today</a>!
```

Comment:
edit link, should be a preview link @vs77bb

Reply:
@owocki in practice this doesn't change anything for readers...right? I will be careful to use preview links if so